src/rules/no-commented-out-tests.ts

```diff
@@ -2,7 +2,7 @@ import { TSESTree } from '@typescript-eslint/experimental-utils';
 import { createRule } from './utils';
 
 function hasTests(node: TSESTree.Comment) {
-  return /^\s*(x|f)?(test|it|describe)(\.\w+|\[['"]\w+['"]\])?\s*\(/m.test(
+  return /^\s*[xf]?(test|it|describe)(\.\w+|\[['"]\w+['"]])?\s*\(/m.test(
```

Comment:
Missed the backslash in front of the opening bracket I think
```suggestion
  return /^\s*[xf]?(test|it|describe)(\.\w+|[['"]\w+['"]])?\s*\(/m.test(
```

Reply:
I don't think so, as that change causes the tests to fail :)

Reply:
Oh I see, the one at the closing bracket can be removed because there's no matching opening bracket for it to close so it is treated as a normal character instead, even without an escape.
But I think it makes the regex hard for humans to parse if one bracket is escaped but its counterpart is not (like how I was confused what was going on :smile:). I think we should leave it as is and just change the part at the front.